conf/test.config

```diff
@@ -38,8 +38,8 @@ params {
     metadata_category_barplot = "treatment1,badpairwise10"
 
     //restrict ANCOM analysis to higher taxonomic levels
-    dada_tax_agglom_max = 4
-    qiime_tax_agglom_max = 4
+    tax_agglom_max = 4
+    tax_agglom_max = 4
```

Comment:
This seems to be a copy paste error?

Reply:
An automated replacement error, yes, will fix!